js/shopping_list_test.js

```diff
@@ -0,0 +1,118 @@
+/* jshint esversion:6*/
+const expect = chai.expect;
+
+describe('Shopping List Item Class', function(){
+  var listItem = ShoppingListItem;
+  var someName = null;
+  before(function(){
+   someName = new ShoppingListItem('carrots', 'a vegetable');
+
+ });
+  it('Shopping list item is a class', function(){
+    expect(listItem).to.be.a('function');
+  });
+  it('Should have a property of name', function(){
+    expect(listItem).to.have.ownPropertyDescriptor('name');
+  });
+  it('Name property should be a string', function(){
+    expect(someName.name).to.be.a('string');
+  });
+  it('Should have a property of description', function(){
+    expect(someName).to.have.ownPropertyDescriptor('description');
+  });
+  it('Description property should be a string', function(){
+    expect(someName.description).to.be.a('string');
+  });
+  it('Should have a property of is_done', function(){
+    expect(someName).to.have.ownPropertyDescriptor('is_done');
+  });
+  it('is_done property should be a boolean', function(){
+    expect(someName.is_done).to.be.a('boolean');
+  });
+  it('Should have a method named check', function(){
+    expect(someName.check).to.be.a('function');
+  });
+  it('Should reassign the value of is_done to true', function(){
+    someName.check();
+    expect(someName.is_done).to.be.true;
+  });
+
+  it('Should have a method named uncheck', function(){
+    expect(someName.uncheck).to.be.a('function');
+  });
+  it('Should reassign the value of is_done to false', function(){
+    someName.uncheck();
+    expect(someName.is_done).to.be.false;
+  });
+  it('Should have a method named render', function(){
+    expect(someName.render).to.be.a('function');
+  });
+  it('Should return a string', function(){
+    expect(someName.render()).to.be.a('string');
+  });
+  it('It should return a li with a class of "completed_is_done"', function(){
+    expect(someName.render()).contains('<li class="completed_' + someName.is_done + '">');
```

Comment:
When checking a contains you likely want to test a specific string that you define. In this case you should not use dynamic text `+ someName.is_done +`.
In a different test that checks that the class is set correctly you can use dynamic text.